### PR TITLE
Add LCEL-compatible `SerpApiRun` and `SerpApiResults` tools

### PR DESCRIPTION
**Pull request by ilyazub**
- **Description:** Add LCEL-compatible `SerpApiRun` and `SerpApiResults` tools
  - **Tag maintainer:** @baskaryan
  - **Twitter handle:** [@ilyazub_](https://twitter.com/ilyazub_)

Example usage:

```python
# docs/docs/expression_language/cookbook/tools.ipynb
search = SerpApiRun()

template = """turn the following user input into a search query for a search engine:

{input}"""
prompt = ChatPromptTemplate.from_template(template)

model = ChatOpenAI()

chain = prompt | model | StrOutputParser() | search

chain.invoke({"input": "Which club is Cristiano Ronaldo playing right now?"})
```

### CODE REVIEW

File: docs/docs/expression_language/cookbook/tools.ipynb
Comment (baskaryan) [inline docs/docs/expression_language/cookbook/tools.ipynb:1]: i think this example would probably be more visible / interesting to add to `docs/docs/integrations/tools/serpapi.ipynb`

**Review by baskaryan — COMMENTED** [review]
nice!

File: docs/docs/expression_language/cookbook/tools.ipynb
Comment (ilyazub) [reply docs/docs/expression_language/cookbook/tools.ipynb:1]: Thanks, sounds good.

**Review by ilyazub — COMMENTED** [review]
@baskaryan I moved the LCEL example to `docs/docs/integrations/tools/serpapi.ipynb` as you suggested.

Thanks for your review!

File: docs/docs/expression_language/cookbook/tools.ipynb
Comment (ilyazub) [reply docs/docs/expression_language/cookbook/tools.ipynb:1]: Done, thank you.

**Pull request closed without merging**